### PR TITLE
Doc 7979 error handling

### PR DESCRIPTION
**Pull request by osfameron**
requires https://github.com/couchbase/docs-sdk-common/pull/39

### CODE REVIEW

File: modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc [hunk 1]
```diff
@@ -263,8 +263,8 @@ There have been changes made in the following areas:
 
 === Exception hierachy
 
-The exception hierachy is now flat and unified under a `CouchbaseException`.
-Each `CouchbaseException` has an associated `ErrorContext` which is populated with as much info as possible and then dumped alongside the stack trace if an error happens.
+The exception hierachy is now flat and unified under a `CouchbaseError`.
+Each `CouchbaseErroreption` has an associated `ErrorContext` which is populated with as much info as possible and then dumped alongside the stack trace if an error happens.
```
Comment (maria-robobug) [inline modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc]: `CouchbaseError`

File: modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc [hunk 2]
```diff
@@ -510,7 +511,7 @@ Parsing of the input failed {"completed":true,"coreId":1,"errors":[{"code":3000,
 	at com.couchbase.client.java.Cluster.query(Cluster.java:225)
 ----
 
-Not only does it throw a `CouchbaseException`, it also tries to map it to a specific exception type and include extensive contextual information for a better troubleshooting experience.
+Not only does it throw a `CouchbaseErroreption`, it also tries to map it to a specific exception type and include extensive contextual information for a better troubleshooting experience.
```
Comment (maria-robobug) [inline modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc]: `CouchbaseError`

**Review by maria-robobug — APPROVED** [review]
Just minor comments, LGTM.

File: modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc (hunk 1 above)
Comment (osfameron) [reply modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc]: hahahahahahahah oh dear, thanks :paddling:

**Pull request merged**